sound/soc/sof/ipc.c

```diff
@@ -31,11 +34,23 @@
 #include "sof-priv.h"
 #include "ops.h"
 
-/* IPC message timeout (msecs) */
-#define IPC_TIMEOUT_MSECS	300
+/* stop PCM on XRUN - used for pipeline debug */
+#define SOF_DEBUG_XRUN_STOP	0
```

Comment:
Should we make this a Kconfig option?

Reply:
Done, and also created a debug option submenu  where stuff like this can go